spec/api/api_version_spec.rb

```diff
@@ -2,7 +2,7 @@
 require 'digest/sha1'
 
 describe 'Stable API warning system', api_version_check: true do
-  API_FOLDER_CHECKSUM = '58773324449147857429086f9ca4a380817bcd96'
+  API_FOLDER_CHECKSUM = '1168d221e57996aae58b9ded56e60062d003e5be'
```

Comment:
the checksum will affect the unit test result, so if we submit it in separate PR, it will lead the failure of one test case